memcached.c

```diff
@@ -8233,10 +8531,39 @@ int main (int argc, char **argv) {
     /* initialize other stuff */
     logger_init();
     stats_init();
```

Comment:
Looks like the logger stats bits can be deleted (and they effectively do nothing here, since it's just double-zeroing stuff). I made the improvement the comment is asking for in cb01d5044d370e2426fddde5bb0e5404b8e7deb0 - going to delete those lines now since that's an obvious fix.